src/index.html

```diff
@@ -5,9 +5,53 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+
+    <article
+      data-qa="card"
+      class="card"
+    >
+      <img
+        class="card__image"
+        src="images/imac.jpeg"
+        alt="imac"
+      >
+
+      <p class="card__text">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</p>
+
+      <p class="card__text-code">Product code: 195434</p>
+
+      <div class="card__review">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+
+        <p class="card__review card__review--score">Reviews: 5</p>
+      </div>
+
+      <div class="card__price card__price--cost">
+        <p class="card__price card__price--cost card__price--cost-text">Price:</p>
+
+        <p class="card__price card__price--cost card__price--cost-number">$2,199</p>
+      </div>
```

Comment:
```suggestion
      <div class="card__price">
        <p class="card__price-text">Price:</p>

        <p class="card__price-number">$2,199</p>
      </div>
```
that would be the correct hierarchy. If an item (child) is nested in another item (parent) it doesn't make him automatically a BEM-element of parent. In other words: `card__price`, `card__price-text`, `card__price-number` are just elements of `card` block. There are no need modificators (use **one** dash `-` for classes with 2 or more words like `price-text`) repeating classes